libmscore/read206.cpp

```diff
@@ -1574,9 +1546,9 @@ static QString ReadStyleName206(XmlReader& e)
 //    before setting anything else.
 //---------------------------------------------------------
 
-static bool readTextPropertyStyle206(XmlReader& e, TextBase* t, Element* be)
+static bool readTextPropertyStyle206(QString xmlTag, XmlReader& e, TextBase* t, Element* be)
```

Comment:
A minor suggestion: to ensure that this `XmlReader` is not actually used for reading it might be good to also declare the `e` parameter as `const XmlReader& e` (this will also require to mark `XmlReader::lookupUserTextStyle()` function with `const` modifier which should logically be there anyway).